draft-ietf-rtcweb-jsep.xml

```diff
@@ -1449,7 +1458,8 @@ candidate:1 1 UDP 1694498815 192.0.2.33 10000 typ host
           and continues until setRemoteDescription is called with
           either a final answer, at which point the PeerConnection can
           fully adopt the pending local description, or a rollback,
-          which results in a revert to the current local
+          which results in a revert back from the pending local
+          description to the current local
```

Comment:
This seems like awkward language. I think I might say
"which causes the JSEP implementation to revert back from the pending local description to the current locatl description"